examples/call-contract-with-token/index.js

```diff
@@ -13,87 +16,119 @@ function generateWalletAddresses(numberOfWallets) {
 
 async function printMultipleBalances(aliases, addresses, tokenContract) {
   for (let i = 0; i < addresses.length; i++) {
-    printBalance(aliases[i], addresses[i], tokenContract);
+    await printBalance(aliases[i], addresses[i], tokenContract);
   }
 }
 
 async function printBalance(alias, address, tokenContract) {
   const balance = await tokenContract.balanceOf(address);
-  console.log(`${alias} has ${ethers.utils.formatUnits(balance, 6)} UST.`);
+  console.log(
+    `${alias} has ${ethers.utils.formatUnits(
+      balance,
+      6
+    )} ${await tokenContract.symbol()}.`
+  );
 }
 
-// Create multiple recipients
-const recipientAddresses = generateWalletAddresses(5);
-const alias = recipientAddresses.map((_, i) => `chain2Recipient${i + 1}`);
-const fundAmount = ethers.utils.parseUnits("1000", 6);
-const sendAmount = ethers.utils.parseUnits("100", 6);
+const cliArgs = process.argv.slice(2);
+const network = cliArgs[0] || "local"; // This value should be either 'local' or 'testnet'
 
+// Create multiple recipients
+const recipientAddresses = generateWalletAddresses(5); // generate random wallet addresses.
+const aliases = recipientAddresses.map(
+  (_, i) => `[Chain B] destination wallet ${i + 1}:`
+); // recipient wallets aliases used for logging.
+const sendAmount = ethers.utils.parseUnits("5", 6); // ust amount to be sent
 
 (async () => {
-  console.log("==== Preparing chain1... ====");
-  const chain1 = await axelar.createNetwork({ seed: "chain1" });
-  const [chain1Sender, chain1Deployer] = chain1.userWallets;
-  console.log("\n==== Preparing chain2... ====");
-  const chain2 = await axelar.createNetwork({ seed: "chain2" });
-  const [chain2Deployer] = chain2.userWallets;
-
-  // Deploy DistributionExecutor contract on the source chain.
-  console.log("\n==== Deploying DistributionExecutor contract on the destination chain... ====");
-  const sourceDistributionContract = await deployContract(
-    chain1Deployer,
-    distributionExecutableAbi,
-    [chain1.gateway.address, chain1.gasReceiver.address]
+  // ========================================================
+  // Step 1: Setup wallet and connect with the chain provider
+  // ========================================================
+  const sourceWallet = new ethers.Wallet(privateKey);
+  const { chainA, chainB } =
+    network === "testnet"
```

Comment:
Lots of code here that conditions on `network`.  This code should be independent of network, similar to https://github.com/axelarnetwork/axelar-local-gmp-examples/blob/b54bc9f94aa815a6603332f3043f29ef0d613753/advanced-examples/testnet/test.js